test/functional/test_framework/test_framework.py

```diff
@@ -113,6 +116,8 @@ def main(self):
         success = TestStatus.FAILED
 
         try:
+            if self.options.usecli and not self.supports_cli:
+                raise SkipTest("--usecli specificed but test does not support using CLI")
```

Comment:
Why `SkipTest`? IMO should fail immediately in the test runner.

Reply:
This allows `--usecli` to be passed to test_runner and any tests that don't support `--usecli` will be skipped:

```
./test_runner.py --usecli
[...]
TEST                           | STATUS    | DURATION

abandonconflict.py             | ○ Skipped | 2 s
bip65-cltv-p2p.py              | ○ Skipped | 2 s
[...]
multiwallet.py                 | ✓ Passed  | 4 s
multiwallet.py --usecli        | ✓ Passed  | 4 s
[...]
zmq_test.py                    | ○ Skipped | 2 s

ALL                            | ✓ Passed  | 140 s (accumulated) 
Runtime: 36 s
```

I have however fixed the typo in the exception message.